packages/workflow/src/internals.ts

```diff
@@ -707,6 +719,11 @@ export class Activator implements ActivationHandler {
       throw new TypeError('Missing activation update protocolInstanceId');
     }
 
+    // If update has __temporal_ prefix but no handler exists, throw error
```

Comment:
We'd need to skip interceptors on built-in updates, same as we did above for queries. We don't have any built-in update anyway, so no need to actually implement this just now, but we should at least leave a trace so we don't forget about this if ever we add one. Just add a comment here. And actually, you could even just remove the ` && !this.updateHandlers.get(name)` part of the condition, as we know it will never evaluate to true unless code around is modified.